content/nb_collectives_intro.tex

```diff
@@ -0,0 +1,31 @@
+An \openshmem nonblocking collective operation, like blocking collective
+operation, is a group communication operation among the
+participants of the team. All participants of the team are required to call the
+collective operation.
+
+\begin{enumerate}
+
+\item Invocation semantics: Upon invocation of a collective routine interface,
+the operation is posted and returned immediately. All participants of the Team
+should call this routine.
+
+\item Collective Types: The nonblocking variants supported include barrier, alltoall,
+and broadcast collectives. Other collective operations such as
+reductions, collect, barrier, alltoalls, and sync will not have nonblocking variants.
+
```

Comment:
Should `sync_all` be supported?  My hunch is it seems possibly useful... but lower priority.